# Request from Client 
bool request
---
# Response from Server
string data
